.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
do we want to specify some configurations rather than leaving this empty?

Reply:
I'd rather stay with the default :)